web_ckeditor4/static/src/js/web_ckeditor4.js

```diff
@@ -214,11 +220,18 @@ openerp.web_ckeditor4 = function(instance)
             this._cleanup_editor();
         }
     });
-    instance.web_ckeditor4.FieldCKEditor4Raw = instance.web_ckeditor4.FieldCKEditor4.extend({
+    var FieldCKEditor4Raw = FieldCKEditor4.extend({
         filter_html: function(value)
         {
             return value;
         }
     });
-}
-
+    core.form_widget_registry.add('text_ckeditor4', FieldCKEditor4);
```

Comment:
I would document in the README what are those widgets and what difference there is between them.